presto-main/src/main/java/com/facebook/presto/operator/scalar/ArrayFindFirstWithOffsetFunction.java

```diff
@@ -136,6 +140,9 @@ public static Slice findSliceUtil(
             }
             Boolean match = function.apply(element);
             if (TRUE.equals(match)) {
+                if (element == null) {
+                    checkCondition(false, INVALID_FUNCTION_ARGUMENT, "FIND_FIRST finds NULL as match, which is not supported. Consider use FIND_FIRST_INDEX function.");
```

Comment:
Consider use -> Consider using

here and in all the error message

Reply:
Done